src/App.css

```diff
@@ -1206,4 +1213,84 @@
     border-top: 1px solid var(--border-color);
     padding-top: 24px;
   }
-} 
\ No newline at end of file
+} 
+
+/* Priority Styles */
+.task-priority-selector {
+  margin-bottom: 1rem;
+  padding: 12px;
+  background: var(--hover-bg);
+  border-radius: 6px;
+}
+
+.task-priority-selector label {
+  display: block;
+  margin-bottom: 8px;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: var(--text-secondary);
+  font-weight: 600;
+  letter-spacing: 0.5px;
+}
+
+.priority-options {
+  display: flex;
+  gap: 8px;
+}
+
+.priority-button {
+  padding: 6px 12px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: all 0.2s ease;
+  font-weight: 500;
+  font-size: 12px;
+  border: 2px solid transparent;
+  flex: 1;
+  text-align: center;
+}
+
+.priority-button:hover {
+  transform: translateY(-1px);
+}
+
+/* Priority Indicator on Task Card */
+.priority-indicator {
+  position: absolute;
+  top: 8px;
+  right: 8px;
+  padding: 2px 8px;
+  border-radius: 12px;
+  font-size: 0.75rem;
+  font-weight: 600;
+  color: white;
+}
+
+.priority-HIGH {
+  background-color: var(--priority-high);
+}
+
+.priority-MEDIUM {
+  background-color: var(--priority-medium);
+  color: #000;
+}
+
+.priority-LOW {
+  background-color: var(--priority-low);
+}
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
.priority-HIGH {
  background-color: var(--priority-high);
  color: var(--text-primary);
}

.priority-MEDIUM {
  background-color: var(--priority-medium);
  color: var(--text-primary);
}

.priority-LOW {
  background-color: var(--priority-low);
  color: var(--text-primary);
}
```
Consider using CSS custom properties for the priority colors to maintain consistency with the existing theme system and enable easier theme switching.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;